js/ui/grid_core/ui.grid_core.data_source_adapter.js

```diff
@@ -393,6 +393,8 @@ module.exports = gridCore.Controller.inherit((function() {
 
                 that._handleDataLoading(loadResult);
                 executeTask(function() {
+                    if(dataSource._disposed) return;
```

Comment:
Maybe better add the `isDisposed` method to dataSource?

```
isDisposed: function() {
  return !!this._isDisposed;
}
```

NB dxPivotGrid's dataSource already has this method, so you can extract it

Reply:
We can replace 
```javascript
dataSource._disposed
```
condition to 
```javascript
!dataSource.store()
```
and does not public isDispose event.